build.sbt

```diff
@@ -31,7 +31,8 @@ val baseSettings = Seq(
     "org.scalatest" %% "scalatest" % "2.2.3" % "test"
   ),
   compilerOptions,
-  coverageExcludedPackages := ".*demo.*"
+  coverageExcludedPackages := ".*demo.*",
+  wartremoverWarnings in (Compile, compile) ++= Warts.all
```

Comment:
You might want to use `Warts.unsafe` here. `NoNeedForMonad` is pretty unstable :(